client/my-sites/checkout/checkout-thank-you/licensing-thank-you-auto-activation-completed.tsx

```diff
@@ -0,0 +1,114 @@
+import { Button } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import { FC, useMemo } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import QueryProducts from 'calypso/components/data/query-products-list';
+import QuerySites from 'calypso/components/data/query-sites';
+import LicensingActivation from 'calypso/components/jetpack/licensing-activation';
+import PageViewTracker from 'calypso/lib/analytics/page-view-tracker';
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
+import {
+	isProductsListFetching as getIsProductListFetching,
+	getProductName,
+} from 'calypso/state/products-list/selectors';
+import getJetpackCheckoutSupportTicketDestinationSiteId from 'calypso/state/selectors/get-jetpack-checkout-support-ticket-destination-site-id';
+import useGetJetpackActivationConfirmationInfo from './use-get-jetpack-activation-confirmation-info';
+
+interface Props {
+	productSlug: string;
+	receiptId?: number;
+	jetpackTemporarySiteId?: number;
+}
+
+const LicensingActivationThankYouCompleted: FC< Props > = ( {
+	productSlug = 'no_product',
+	jetpackTemporarySiteId = 0,
+} ) => {
+	const translate = useTranslate();
+	const dispatch = useDispatch();
+
+	const hasProductInfo = productSlug !== 'no_product';
+
+	const productName = useSelector( ( state ) =>
+		hasProductInfo ? getProductName( state, productSlug ) : null
+	);
+
+	const isProductListFetching = useSelector( ( state ) => getIsProductListFetching( state ) );
+
+	const destinationSiteId = useSelector( ( state ) =>
+		getJetpackCheckoutSupportTicketDestinationSiteId( state, jetpackTemporarySiteId )
+	);
+	const automaticTransferSucceeded = destinationSiteId > 0;
+
+	const title = useMemo( () => {
+		return automaticTransferSucceeded
```

Comment:
It worries me a little bit that people in the future might confuse `automaticTransferSucceeded` with an automated transfer used in the context of Atomic sites. Can we add a comment to specify that this is a different kind of transfer?